### DISCUSSION TITLE
Configure Vertx Option

### DISCUSSION THREAD

**Discussion by h2mch** [smallrye/smallrye-reactive-messaging#2725] [Q&A] [answered]
Is it possible that we can configure the underlining [vertx option](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html).

Specially I'm interested to increase the [DEFAULT_MAX_EVENT_LOOP_EXECUTE_TIME](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#DEFAULT_MAX_EVENT_LOOP_EXECUTE_TIME) from 2S to something more.

I'm using smallrye-reactive-messaging with a kafka-connector within wildfly.

**Comment by cescoffier** [comment]
That's a request for @kabir, as reactive messaging is implemented in a way that uses the Vert.x instance provided by the platform (Wildfly in your case). If none is provided, it creates a default one - but it's up to the integration to provide the configured one.

**Comment by kabir** [comment, marked as answer]
Well look into this on the WildFly side https://lists.jboss.org/archives/list/wildfly-dev@lists.jboss.org/thread/CHUJ5GZSHXUSVZNXS3X32H7UQ4M64NKC/
